pages/innofest/index.tsx

```diff
@@ -80,23 +86,142 @@ export default function InnoFest(props: LandingBodyProps) {
+                <Text fontFamily="Segoe Script" fontSize={{ base: "md", lg: "xl" }}><i>Innofest 22/23 Presents</i></Text>
+                <Text fontFamily="Century Gothic" className="glowtext1">SOLVATHON</Text>
+              </Heading>
+              <Text fontSize={{ base: "md", lg: "2xl" }} mb={5}>
+                The ideas of our innovators have solved many pain points of past.
+              </Text>
+              <Grid
+                templateColumns={{
+                  base: "repeat(2, 1fr) ",
+                  xl: "repeat(3, 1fr)",
+                }}
+                gap={3}
+                backgroundColor={primary[800]}
+                mb={18}
+                position="relative"
+              >
+                <GridItem w="75%" key={0} margin="auto">
+                    <Image align="center" src={"https://i.gyazo.com/9da0564863849505a6f069d2b48dc820.png"} width="100%" alt="" />
+                    <Text fontSize={{ lg: "md" }} align="center">Digitising paper processes: MT Fleet App</Text>
+                </GridItem>
+                <GridItem w="75%" key={1} margin="auto">
+                    <Image align="center" src={"https://i.gyazo.com/992a564c04fcb42a217d4d7fb007eb4b.png"} width="100%" alt="" />
+                    <Text fontSize={{ lg: "md" }} align="center">Saving Manpower: Wingtip Sensors</Text>
+                </GridItem>
+                <GridItem w="75%" key={2} margin="auto">
+                    <Image align="center" src={"https://i.gyazo.com/6a703c5603bb680d393def72094effc3.png"} width="100%" alt="" />
+                    <Image align="center" src={"https://i.gyazo.com/bc8cd22c67704ed00f3f4a933178a361.png"} width="100%" alt="" />
+                    <Text fontSize={{ lg: "md" }} align="center">No more scratching or removal: CDA Sticker</Text>
+                </GridItem>
+              </Grid>
+              <Text fontSize={{ base: "md", lg: "2xl" }} mb={5}>
+                In our efforts to <b><i>#makeinnovationjoyful</i></b>, <Text as="span" color={nYellow[400]}>SW<i>i</i>FT</Text>{" "} wants to hear your pain
+                point AND partner you to solve the pain point through innovation!
+              </Text>
+              <Box
+              text-align="justify"
+              py={{ base: 10, md: 5 }}
+              px={{ base: 5, md: 5 }}
+              boxShadow={"base"}
+              rounded={"3xl"}
+              backgroundColor={primary[600]}
+              w="100%"
+              id="solvathon"
+              mb={5}
+              >
+                <Heading size={"lg"} mb={5} color={nYellow[400]} className="glowtext3">
+                  What you need to do?
+                </Heading>
+                <List spacing={3} >
+                  <ListItem>
+                    <ListIcon as={ChevronRightIcon} color={nGreen[400]} />
+                    Bookmark <a href="http://localhost:3000/innofest">this URL</a> or join our SW<i>i</i>FT Telegram Channel.
```

Comment:
```suggestion
                    Bookmark <Link href="/innofest">this URL</Link> or join our SW<i>i</i>FT Telegram Channel.
```
the localhost url will not work for prod and stag env where the url is different, consider using mui's `Link` component as well

Reply:
Done with this